index.js

```diff
@@ -1,60 +1,69 @@
 'use strict'
 
 const { Client } = require('discord-rpc');
+const julian = require('julian');
+const moment = require('moment');
 
-const clientId = '811111315988283413';
-
+const clientId = '811145575134658561';
 const day = 24 * 60 * 60;
 const landingTimestamp = 1613681692;
 
+function getMSD(earthTime) {
+    earthTime = moment(earthTime).toDate();
+    return (julian(earthTime) - 2405522.0028779) / 1.0274912517;
+};
```

Comment:
```suggestion
function getMSD(earthTimestamp) {
  return (julian(earthTimestamp) - 2405522.0028779) / 1.0274912517;
};
```
There's no reason to actually use moment here, we're using timestamps already!